wp-content/themes/core/assets/pcss/media/_mixins.pcss

```diff
@@ -17,7 +17,7 @@
 
 		/* Fill available space left/right of contentSize block */
 		@media (--viewport-large) {
-			max-width: calc(((100% - var(--wp--style--global--content-size)) / 2) - 2em);
+			max-width: 100%;
```

Comment:
@GeoffDusome does the Moose styleguide in Figma match this implementation? If not, would you leave a note for Sarah to update in Figma?